pagarme/former-kit-skin-pagarme
# Update dependencies

## Context

Upgrade all dependencies, and fix interfaces problems.

## Checklist
- [x] Update major dependencies
- [x] Refactor dependencies usage


## Linked Issues
- [x] resolves #154 


## How to test this
1 - Download the `former-kit` branch from PR https://github.com/pagarme/former-kit/pull/217
2 - Go to the `former-kit` project folder, delete `yarn.lock` and `node_modules`(just in case 😄 )
3 - Download the branch from this PR, go to its folder and build it using:
```
yarn build
```
4 - Create a link using the following command
```
yarn link
```
5 - Back to the `former-kit` folder
6 - Link the `former-kit-skin-pagarme` in the `former-kit` using
```
yarn link former-kit-skin-pagarme
```
7 - Install all dependencies using the following command
```
yarn
```
8 - Start the _Storybook_ process using:
```
yarn storybook
```

The _Storybook_ should run without warnings.
